signalfx-tracing/signalfx-nodejs-tracing/koa/server.js

```diff
@@ -0,0 +1,38 @@
+if (process.env.NODE_ENV !== 'production') {
+  require('dotenv').config();
```

Comment:
Is this used?  I don't see a config file.

Reply:
Updated. Was for the previous API I used